tests/failpoints/cases/test_transaction.rs

```diff
@@ -16,3 +17,13 @@ fn test_txn_failpoints() {
     must_commit(&engine, k, 10, 20);
     fail::remove("commit");
 }
+
+#[test]
+fn test_pipelined_txn_failpoints() {
+    fail::cfg("pipelined_txn_write_finished", "delay(200)").unwrap();
```

Comment:
I do not understand why we delay in this fail-point. This `fail-point` is in `callback`, so it will only delay the `apply-thread`, which call this callback after writing into RocksDB.

Reply:
The purpose is to test storage::Scheduler could handle different events order correctly:
1, `pipelined_txn_return_to_caller` happens first, then `pipelined_txn_write_finished`
2,  `pipelined_txn_write_finished` happens first, then `pipelined_txn_return_to_caller` 
Both could happen, the `delay` simulate these cases.